.github/workflows/gh-pages.yaml

```diff
@@ -65,7 +67,7 @@ jobs:
           echo "URL=https://$GITHUB_REPOSITORY_OWNER.github.io/${GITHUB_REPOSITORY##*/}/$FOLDER" >> $GITHUB_OUTPUT
 
       - name: Deploy to GitHub Pages
-        uses: peaceiris/actions-gh-pages@v3
+        uses: peaceiris/actions-gh-pages@v3.9.3
```

Comment:
We don't need to. This action when they release a new version will push to v3 tag

Reply:
ok, I'm trying to find why we have 404 on every ghpages deployed. do you have any idea?

Reply:
It need to finish the 2nd action to deploy a new folder: https://github.com/linagora/twake-on-matrix/actions/runs/6417229541

If too many PRs are opened this action will be cancelled, so some PR might not have it

Reply:
oh ok I get it tyvm